src/core/getFrameHtmlResponse.ts

```diff
@@ -19,8 +19,8 @@ function getFrameHtmlResponse({
   refreshPeriod,
   refresh_period,
 }: FrameMetadataType): string {
-  // Set the image metadata if it exists.
-  const imageHtml = image ? `  <meta property="fc:frame:image" content="${image}" />\n` : '';
+  const imageHtml = `  <meta property="fc:frame:image" content="${image}" />\n`;
+  const ogImageHtml = `  <meta property="og:image" content="${image}" />\n`;
```

Comment:
From my testing, `og:title` is also required to have warpcast render the frame. 
Can just set to empty string to satisfy this.

Reply:
Or, looks like they have some updated docs, and it seems maybe not necessary: https://docs.farcaster.xyz/reference/frames/spec#required-properties

Reply:
Not sure i follow your point

Reply:
You added the `og:image` metadata here, but not `og:title`.

From my testing, without `og:image` AND `og:title`, warpcast won't render a frame from a GET request. 

This works:
```html
<!DOCTYPE html>
<html>
    <head>
	<meta property="og:image" content="https://paperclip.a.pinggy.link/images/nouns-auction-house.png" />
	<meta property="og:title" content="title" />
	<meta property="fc:frame" content="vNext" />
	<meta property="fc:frame:image" content="https://paperclip.a.pinggy.link/images/nouns-auction-house.png" />
	<meta property="fc:frame:post_url" content="https://paperclip.a.pinggy.link/auction/nouns?" />
	<meta property="fc:frame:button:1" content="View auction!" />
	<meta property="fc:frame:button:1:action" content="post" />
    </head>
</html>
    
```

This doesn't work (only difference is that it is missing the `og:title` tag)
```html
<!DOCTYPE html>
<html>
    <head>
	<meta property="og:image" content="https://paperclip.a.pinggy.link/images/nouns-auction-house.png" />
	<meta property="fc:frame" content="vNext" />
	<meta property="fc:frame:image" content="https://paperclip.a.pinggy.link/images/nouns-auction-house.png" />
	<meta property="fc:frame:post_url" content="https://paperclip.a.pinggy.link/auction/nouns?" />
	<meta property="fc:frame:button:1" content="View auction!" />
	<meta property="fc:frame:button:1:action" content="post" />
    </head>
</html>
```

Reply:
But the spec doesn't care about the title?!?

Is it the spec wrong?

Reply:
Yea which is why I was confused, but I just retested.

It seems the spec is not accurate here. `og:title` definitely makes a difference.

Reply:
Let me loop back this to V

Reply:
But yeah, if it doesn't work, than yeah let me add `og:title`, and thank you for triple checking this.

Reply:
Awesome, thanks!

Oh, and `content=""` (empty string) doesn't work lol (very finicky), so just put "title" or something...

Reply:
Sorry for the nits lol

This is the last thing stopping us from moving over to use onchainkit 🙏!

Reply:
🤔 I guess is fair to let people decide, and than yeah maybe the default is something like `Frame title`